templates/account/details.html

```diff
@@ -152,6 +134,28 @@ <h3 class="text-sm-center">
         </div>
       </div>
       <div class="tab-pane fade" id="settings" role="tabpanel">
+        {% if user.email %}
```

Comment:
I think it can never be false, as we always have the user's email. 

There's one case, where `user.password` might be false and it's when one signed up via Google/Facebook social login